cosmos/kima_network.json

```diff
@@ -0,0 +1,55 @@
+{
+    "chainId": "kima_network",
+    "chainName": "Kima Network",
+    "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/kimanetwork/chain.png",
+    "rpc": "https://rpc.kima.network/",
+    "rest": "https://api.kima.network/",
+    "nodeProvider": {
+      "name": "Kima Network",
+      "email": "support@kima.network",
+      "website": "https://kima.network/"
+    },
+    "bip44": {
+      "coinType": 118
+    },
+    "bech32Config": {
+      "bech32PrefixAccAddr": "kima",
+      "bech32PrefixAccPub": "kimapub",
+      "bech32PrefixValAddr": "kimavaloper",
+      "bech32PrefixValPub": "kimavaloperpub",
+      "bech32PrefixConsAddr": "kimavalcons",
+      "bech32PrefixConsPub": "kimavalconspub"
+    },
+    "currencies": [
+      {
+        "coinDenom": "KIMA",
+        "coinMinimalDenom": "uKIMA",
+        "coinDecimals": 6,
+        "coinGeckoId": "kima"
+      }
+    ],
+    "feeCurrencies": [
+      {
+        "coinDenom": "KIMA",
+        "coinMinimalDenom": "uKIMA",
+        "coinDecimals": 6,
+        "coinGeckoId": "kima",
+        "gasPriceStep": {
+          "low": 0.001,
+          "average": 0.0025,
+          "high": 0.004
+        }
+      }
+    ],
+    "stakeCurrency": {
+      "coinDenom": "KIMA",
+      "coinMinimalDenom": "uKIMA",
+      "coinDecimals": 6,
+      "coinGeckoId": "kima"
+    },
+    "features": ["ibc-transfer"],
+    "additionalEndpoints": {
+      "archive": "https://archive.kima.network/genesis.json",
+      "grpc": "https://grpc.kima.network/"
```

Comment:
The unknown fields should be removed.

Reply:
Hi!
1. Should I submit a new PR with the image folder names changed?
2. Which unknown fields should be removed? I think we provided values for all the fields in the JSON files?

Reply:
1. No, you shouldn't. Just push a revised commit for this PR.
2. The validation doesn't allow unknown fields such as `additionalEndpoints`.